src/create-element.js

```diff
@@ -93,7 +93,9 @@ export function coerceToVNode(possibleVNode) {
 
 	// Clone vnode if it has already been used. ceviche/#57
 	if (possibleVNode._dom!=null) {
-		return createVNode(possibleVNode.type, possibleVNode.props, possibleVNode.text, possibleVNode.key, null);
+		let vnode = createVNode(possibleVNode.type, possibleVNode.props, possibleVNode.text, possibleVNode.key, null);
```

Comment:
Can we add `_dom` as an argument to createVNode here? I think it might be shorter (could totally be wrong!)

Reply:
Tried that and adding the argument to `createVNode` makes the PR bigger by `+3 B` :S